src/test/java/com/anusha/testing/AppTest.java

```diff
@@ -42,4 +51,318 @@ public void shouldMultiplyTwoIntegers(){
+
+    // Integer actualLowest = myArr.get( myArr.size() - ) ; ;
+    int actualHighest = myArr.get( myArr.size() - 1) ;
+    int actualLowest = myArr.get( 0 ) ;;
+    int expectedLowest=9;
+    int expectedHighest=98;
+
+    Assert.assertEquals(expectedLowest,actualLowest);
+    Assert.assertEquals(expectedHighest,actualHighest);
+
+}
+/*
+ @Test
+    public void  shouldCheckElementsCopied(){
+     ArrayList<Integer> arrayList1 = new ArrayList<>();
+
+     arrayList1.add(100);
+     arrayList1.add(200);
+     arrayList1.add(300);
+
+     ArrayList arrayList2 = new ArrayList();
+
+     arrayList2.add(400);
+     arrayList2.add(500);
+     arrayList2.add(600);
+     arrayList2.add(700);
+     arrayList2.add(800);
+
+List<Integer> actual=ArrayUtils.getCopiedElementsIntoAnotherList();
+List<Integer> expected=
+        assertThat(actual.getList(), containsInAnyOrder(expectedList.toArray(new String[expectedList.size()]));
+
+ }  */
+
+ @Test
+    public void shouldCheckForStringListLength(){
+     List<String> myString=new ArrayList<>();
+     myString.add("Anusha");
+     myString.add("Gopal");
+     myString.add("Goutham");
+     myString.add("Sriraksha");
+
+     int actual=myString.size();
+     int expected=4;
+
+     Assert.assertEquals(expected,actual);
+ }
+
+ @Test
+    public void shouldCheckForAddedString(){
+     String a="Goutham";
+     String b="Rao";
+     String c=a+b;
+
+     String actual=c;
+     String expected="GouthamRao";
+
+     //Assert.assertEquals(expected,actual);
+     assertThat(c, is("GouthamRao"));
+ }
+
+ @Test
+    public void shouldDisplayUppercasedString(){
+
+     String str="anusha";
+     String actual=str.toUpperCase();
+     String expected="ANUSHA";
+     assertEquals(expected,actual);
+
+ }
+
+ @Test
+ public void shouldDisplayParsedIntegerValue(){
+     int number=2;
+     Integer actual=Integer.valueOf(number);
+     Integer expected=2;
+     assertEquals(expected,actual);
+ }
+
+ @Test
+    public void shouldCheckElementPresentInList(){
+
+     String[] myStr=new String[]{"Goutham","SriRaksha","Anusha","Gopal"};
+     List<String> myList=Arrays.asList(myStr);
+
+     boolean actual=myList.contains("Gopal");
+    assertTrue(actual);
+
+    boolean actuals=myList.contains("Aditya");
+    assertFalse(actuals);
+ }
+
+@Test
+    public void shouldReturnSecondHighestElementFromList(){
+Integer[] array=new Integer[]{100,12,5,45,78};
+List<Integer> myArray=Arrays.asList(array);
+    int secondLargest = myArray.get(0);
+    int largest = myArray.get(0);
+    for (int i = 0; i < myArray.size(); i++)
+    {
+        if(myArray.get(i) > secondLargest)
+        {
+            if(myArray.get(i) > largest ) {
+                secondLargest = largest;
+                largest = myArray.get(i);
+            } else {
+                secondLargest = myArray.get(i);
+            }
+        }
+    }
+  Integer actual=largest;
+    Integer expected=78;
```

Comment:
easiest would be to sort the list by assending order and get the last but one element ....